### DISCUSSION TITLE
Unable to this running

### DISCUSSION THREAD

**Discussion by WoodyDark** [vuepress/core#199] [Q&A] [answered]
I'm using Vue 3.0, but can't seem to get this to work.

I'm using Vite as my bundler and I get the following error when I start the vuepress dev server.

```
clientAppSetup.js:4 Uncaught ReferenceError: Cannot access 'defineClientAppSetup' before initialization
    at clientAppSetup.js:4
```

Relevant versions
```
"vue": "^3.0.0",
...,
"vuepress": "^2.0.0-beta.17",
"vuepress-vite": "^2.0.0-beta.17"
```

My folder structure
```
├── docs
|   ├── .vuepress
|   ├── README.md
├── package.json
```

When I use webpack, I get this error:
```
- vue@3.1.1 (/path/to/node_modules/vue/index.js)
- vue-template-compiler@2.6.13 (/path/to/node_modules/vue-template-compiler/package.json)
```

Any idea how to solve this?

**Comment by Mister-Hope** [comment, marked as answer]
The vue-template-compiler is obviously for vue2. New package is under `@vue` scope.
